src/lib/routing-tables.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+const routing = require('five-bells-routing')
+
+/**
+ * Maintain two sets of `routing.RoutingTables`.
```

Comment:
Despite having the same name, the `RoutingTables` is a bit different from the one in five-bells-routing.